cpp/msbuild/ZeroC.Ice.Cpp.props

```diff
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) ZeroC, Inc. All rights reserved. -->
+<!-- Copyright (c) ZeroC, Inc. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
```

Comment:
Previously the Ice package defined this versions to let the builder package know what Ice version was being used. Know the tools are shipped in the same package and there is no use for this.